packages/tpetra/core/test/FECrsMatrix/FECrsMatrix_UnitTests.cpp

```diff
@@ -227,6 +224,19 @@ std::vector<std::vector<Scalar> > generate_fem1d_element_values() {
   return mat;
 }
 
+template<class Scalar, class Node>
+Kokkos::View<Scalar*[2], Kokkos::LayoutLeft, typename Node::device_type > generate_fem1d_element_values_kokkos() {
```

Comment:
You know that it's a 2x2 matrix, so why not give it the type `Kokkos::View<Scalar[2][2], ...>`?

Reply:
Yes, that's a good idea.